src/main/java/seedu/us/among/logic/commands/CommandResult.java

```diff
@@ -66,6 +66,15 @@ public CommandResult(String feedbackToUser, Endpoint endpoint) {
                 false, false, false, true);
     }
 
+    /**
```

Comment:
In the future, it would be better to use a static method that constructs CommandResult because I think they are a lot of permutations of CommandResult already XD

But since the project is coming to a close, I think this implementation is good enough.

Reply:
yea, was planning to refactor this